config/space-presets.json

```diff
@@ -48,7 +48,7 @@
     }
   },
   {
-    "id": "5apps",
+    "id": "xmpp-5apps",
```

Comment:
Ember is using the id to generate the URLs. I don't know exactly what was going on in detail, but when the preset uses "5apps" as id, it links channels to `/5apps/channel/<channelslug>` but then would error when trying to navigate to it, because it wouldn't find the space by that id.

Using the id based on the name fixes that.